### ISSUE TITLE
Error: When building cache: file tree has path errors

### ISSUE THREAD

**Issue by SuperJC710e** [wagoodman/dive#258]
Hi, thanks for the project. I just built the latest git from source (needed to do so to overcome another issue) and I'm still getting another error. Like this:

```
Building cache...
  path error at layer index Index(0-3:4): unable to remove '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-4:5): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-5:6): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-6:7): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-7:8): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-8:9): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-9:10): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-10:11): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-11:12): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-12:13): unable to add '/.dockerenv': path does not exist: /.dockerenv
  path error at layer index Index(0-13:14): unable to add '/.dockerenv': path does not exist: /.dockerenv
file tree has path errors
```

**Comment by wagoodman** [comment]
I recently added more aggressive error checking with is why you may be seeing this. Do you have a public docker image or a dockerfile too reproduce this?

**Comment by SuperJC710e** [comment]
Hi @wagoodman sorry for the delay. 

I do not have a public image of my lightly modified version, but I was working on a image based on this: `binhex/arch-delugevpn`. I was trying to see why the image was turning out so large. (I was adding a python application called `flexget` to the image.)

That said, I do have a private repo of it hosted on `gitlab`, I could give you an invite should you wish.

**Comment by wagoodman** [comment]
It appears that `2793e4c1aab8b47968fa43c136926f1efd32e1cea4c690d4e8f3493949dec99d/layer.tar` attempts to remove a file that doesn't exist in lower layers (`.dockerenv`) by providing a whiteout file (`.wh..dockerenv`). It seems that there may be something wrong with the tar as well:
```
❯ tar -xvf layer.tar
...
tar: Error exit delayed from previous errors.
```

I'll add a `--ignore-errors` option to dive such that it will do a "best effort" in trying to show the UI and analysis results even if problems like these arise.

**Comment by bitsofinfo** [comment]
i'm experiencing the same thing

**Comment by bitsofinfo** [comment]
over and over 
```  
path error at layer index Index(0-28:29): unable to add '/bin/rpm': path does not exist: /bin/rpm
  path error at layer index Index(0-28:29): unable to add '/lib/systemd': path does not exist: /lib/systemd
  path error at layer index Index(0-28:29): unable to add '/sbin/chkconfig': path does not exist: /sbin/chkconfig
```

**Comment by wagoodman** [comment]
The `--ignore-errors` CLI option and config setting has been merged, once v0.10.0 is release you should be able to get around images with parse issues.

**Comment by asafkin** [comment]
no v0.10.0 yet?
